pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [tool.ruff]
+line-length = 100
```

Comment:
I think we should do this in a different MR

Reply:
Oops, updated by mistake. I'll include in a separated one to avoid some warning when running ruff